src/Forms/Checkbox.test.js

```diff
@@ -119,6 +119,16 @@ describe('<Checkbox />', () => {
             ).toContain('wonderful-styles');
         });
 
+        test('should set validationClassName on the popover', () => {
```

Comment:
> should set childrenClassName on the popover

Reply:
Oh that is super weird. I was sure I changed the title of this test, thanks!